src/Bot.hs

```diff
@@ -485,6 +485,7 @@ builtinCommands =
                     [ ("link", setNoTrustLinkReplyCommand)
                     , ("command", setNoTrustCommandReplyCommand)
                     ])
+              , ("dubtrack", setDubtrack)
```

Comment:
This should be a subcommand of `!config`

Reply:
It is! Just not nested under reply

Reply:
@jappeace ah, my bad! I see it now, sorry. :)